Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN go mod tidy
 RUN go mod vendor
 ARG TEST="true"
 RUN if [ "$TEST" != "false" ]; then ./validate.sh ; fi
-RUN go build -mod=vendor -ldflags "-X github.com/prebid/prebid-server/version.Ver=`git describe --tags | sed 's/^v//'` -X github.com/prebid/prebid-server/version.Rev=`git rev-parse HEAD`" .
+RUN go generate modules/modules.go && \
```

Comment:
Not necessary if we keep it included in source control. Please see my other comment.

Reply:
Agree.